src/string.ts

```diff
@@ -15,9 +15,13 @@ export const capitalize = (str: string): string => {
  *
  * camel('hello world')   -> 'helloWorld'
  * camel('va va-VOOM') -> 'vaVaVoom'
+ * camel('helloWorld') -> 'helloWorld'
  */
 export const camel = (str: string): string => {
-  const parts = str?.split(/[\.\-\s_]/).map(x => x.toLowerCase()) ?? []
+  const parts = str
+    ?.replace(/([A-Z])+/g, capitalize)
```

Comment:
I did not know you could use a function as the replacement argument 🤯 ❤️

Reply:
yes, that's pretty cool, in my service we use it this way with maps, filters, etc... It makes the code more readable